playbooks/update.yml

```diff
@@ -0,0 +1,8 @@
+- name: mediawiki123-apb provision
```

Comment:
Should this be `mediawiki123-apb update`?

Reply:
yes. thank you